src/applications/vaos/components/EnrolledRoute.jsx

```diff
@@ -24,7 +35,8 @@ export default function EnrolledRoute({ component: RouteComponent, ...rest }) {
       verify={!environment.isLocalhost()}
     >
       <Route {...rest}>
-        {!hasRegisteredSystems && <NoRegistrationMessage />}
+        {/* TODO: Delete 'NoRegistrationMessage' component after code review confirmed */}
```

Comment:
I will delete after initial review. Just wanted to make sure I'm on the right path.

Reply:
Ah is this because we are using redirect to address https://github.com/department-of-veterans-affairs/va.gov-team/issues/101532?

Reply:
yes.

Reply:
@simiadebowale - Is it your understanding too that any notifications, warnings, etc. will be displayed to the user when redirected to the my-health page when not LOA3?

Reply:
@vbahinwillit  - yes that's my understanding. This LGTM